TumourModelParticleFiltering/TumourModel_ParticleFiltering.m

```diff
@@ -33,19 +33,21 @@
 
 page = 2; % we are not going to fit this parameter but we need to set it
 
+obsSigma = sqrt(2*2500);
```

Comment:
We have

```
>> nanstd(Control_tumour_growth')

ans =

   1.0e+03 *

  Columns 1 through 10

    0.0162    0.0164    0.0219    0.0397    0.0584    0.0732    0.0893    0.1316    0.1198    0.1616

  Columns 11 through 20

    0.1197    0.2036    0.2182    0.3245    0.4726    0.6744    0.7568    1.0483    1.1231    1.1054

  Columns 21 through 30

    0.9931    0.9690    0.9038    0.8325    0.8632    0.7492    0.6366    0.6820    0.5506    0.4690

  Columns 31 through 33

    0.4198    0.2860    0.2972
```

So `obsSigma = sqrt(2 * 2500) = 70.7107` seems a bit high for early observations but a bit low for later observations so maybe it's about right. Could we estimate it? I am not sure. Maybe we could with particle metropolis hastings but I'd have to go and think hard about it. For now, I think we can stick with value I chose. @adriannejnner what do you think?